extensions/2.0/Vendor/EXT_feature_metadata/examples/weather/weather.gltf

```diff
@@ -0,0 +1,219 @@
+{
+  "asset": {
+    "version": "2.0"
+  },
+  "extensionsUsed": [
+    "EXT_feature_metadata"
+  ],
+  "extensions": {
+    "EXT_feature_metadata": {
+      "extras": {
+        "draftVersion": "0.0.0"
+      },
+      "featureClasses": {
+        "weather": {
+          "name": "Weather Data",
+          "properties": {
+            "airTemperature": {
+              "valueType": "FLOAT32"
+            },
+            "airPressure": {
+              "valueType": "FLOAT32"
+            },
+            "windVelocity": {
+              "valueType": "FLOAT32",
+              "elementType": "ARRAY",
+              "valueCount": 3
+            }
+          }
+        },
+        "town": {
+          "name": "Town Data",
+          "properties": {
+            "name": {
+              "name": "Name",
+              "valueType": "STRING"
+            },
+            "population": {
+              "name": "Population",
+              "valueType": "FLOAT32"
+            }
+          }
+        }
+      },
+      "featureCollections": {
+        "weatherCollection": {
+          "featureClass": "weather",
+          "featureCount": 1000,
+          "properties": {
+            "airTemperature": {
+              "elementCount": 1000,
```

Comment:
@lilleyse and I decided to move element count up to be at the same level as `featureCount`. Having variable-length array sizes isn't very practical and probably error-prone.

Reply:
Just a thought - should `elementCount` be optional and only specified if `featureCount` != `elementCount`?

Reply:
Sounds good with me, pretty often both are the same.

I'll have to look up how to express this in JSON Schema

Reply:
I ended up just describing it as "if not specified, elementCount will be assumed to equal featureCount"